src/mixins/name-request-mixin.ts

```diff
@@ -121,29 +115,23 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
     }
 
     // If the NR's root state is not APPROVED or CONDITIONAL, it is not consumable.
-    if (nr.state !== NameRequestStates.APPROVED && nr.state !== NameRequestStates.CONDITIONAL) {
+    // EXPIRED or CONSUMED should not return NOT_APPROVED.
+    if (![NameRequestStates.APPROVED, NameRequestStates.CONDITIONAL,
```

Comment:
Is the change to `NameRequestStates.CONDITIONAL` from `! NameRequestStates.CONDITIONAL` intentional in all the prs? 
Just wanted to confirm as i've noticed in each.

Reply:
It still says `!`. I just wrapped it in an array

Reply:
Sweet! 
I didn't see the `[]` honestly. Haven't had my coffee yet! 
Will resolve the comment in the other pr's as well.